css/style.css

```diff
@@ -0,0 +1,485 @@
+  width: 18px;
+  height: 18px;
+  background: #152fa1;
+  transform: rotate(45deg);
+}
+
+.level::before {
+  display: none;
+  content: '';
+  position: absolute;
+  top: 20px;
+  right: 102%;
+  width: 18px;
+  height: 18px;
+  background: #D62B2E;
+  transform: rotate(45deg);
+}
+
+.level:hover::before {
+  display: block;
+}
+
+.first-card {
+  position: absolute;
+  left: 60%;
+  top: 35%;
+  transform: rotate(11.9deg);
+  z-index: -2;
+}
+
+.second-card {
+  position: absolute;
+  left: 69%;
+  top: 45%;
+  transform: rotate(39.25deg);
+  z-index: -1;
+}
+
+.hide {
+  display: none;
+}
+
+.table {
+  display: none;
+  /* width: 700px; */
+  width: 100%;
+  /* align-items: start; */
+}
+
+.table-0 {
+  width: 720px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.table-1 {
+  display: flex;
+  height: 94%;
+  max-height: 720px;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.table-2 {
+  display: flex;
+  justify-content: center;
+  width: 600px;
+  height: 95%;
+  flex-wrap: wrap;
+}
+
+
+
+
+.card-wrapper {
+  outline: 2px solid blue;
+  /* width: 31%; */
+  width: 218px;
+  height: 341px;
+  text-align: center;
+  perspective: 1200;
+  -webkit-perspective: 1200;
+  -moz-transform: perspective(1200px);
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d; 
+}
+
+.card-inner {
+  outline: 2px solid red;
+  position: relative;
+  width: 100%;
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d;
+  transition: .7s linear;  
+  -webkit-transition: .7s linear;
+  -moz-transition: .7s linear;
+  -o-transition: .7s linear;
+}
+
+.front-side, .back-side {
+  background-size: contain;
+  background-repeat: no-repeat;
+  outline: 2px solid yellow;
+  width: 100%;
+  /* width: 218px; */
+  height: 341px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  -webkit-backface-visibility: hidden;
+  -moz-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+.back-side {
+  /* width: 100%; */
+  background-image: url(../img/back-side.png);
+  -webkit-transform: rotateY(0deg);
+  -moz-transform: rotateY(0deg);
+  -ms-transform: rotateY(0deg);
+  transform: rotateY(0deg);
+}
+
+.front-side {
+  background-image: url(../img/game-over.png);
+  /* background-size: contain; */
+  /* background-repeat: no-repeat; */
+  -webkit-transform: rotateY(-180deg);
+  -moz-transform: rotateY(-180deg);
+  -ms-transform: rotateY(-180deg);
+  transform: rotateY(-180deg);
+}
+
+.front-side--win {
+  background-image: url(../img/winner.png)
+}
+
+.flip {
+  -webkit-transform: rotateY(180deg);
+  -moz-transform: rotateY(180deg);
+  -ms-transform: rotateY(180deg);
+  transform: rotateY(180deg);
+}
+
+.table-2 .card-wrapper {
+  width: 26%;
+}
+
+.table-2 .card-inner {
+  width: 99%;
+}
+
+.table-2 .card, .table-2 .result {
+  width: 93%;
+}
+
+@keyframes rotation {
+  from {transform: scale(1);}
```

Comment:
Codacy found an issue: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)